composer.json

```diff
@@ -68,7 +68,7 @@
     },
     "scripts": {
         "ci:php:lint": "find *.php Classes Configuration Tests -name '*.php' -print0 | xargs -0 -n 1 -P 4 php -l",
-        "ci:php:codestyle": "php-cs-fixer fix --config .php_cs.dist -v --dry-run --using-cache false --diff --diff-format=udiff",
+        "ci:php:codestyle": "php-cs-fixer fix --config .php_cs.xml -v --dry-run --using-cache false --diff --diff-format=udiff",
```

Comment:
This file is PHP, not XML. Hence, the name needs to be `.php_cs.php`.

Reply:
Sorry, late night changes :) fix pushed